### DISCUSSION TITLE
multiple objects are selected  error

### DISCUSSION THREAD

**Discussion by pingbaohua** [eclipse-theia/theia#14064] [Q&A]
When I use the oracle Developer Tools foe vscode plugin, what is the error when I right-click the node? Could this be the cause of the mismatch between the two versions：

error info: 'Rename' is not supported when multiple objects are selected in Oracle Explorer > Database view.

**Comment by msujew** [comment]
Hey @pingbaohua,

we don't have any influence on what the oracle extension is doing. Do you get the same error if you use the extension in VS Code? If not, please create an issue in this repo with the appropriate reproduction steps and any kind of information you can provide (screenshots, logs, etc.).

**Reply by pingbaohua** [reply]
I tried again to Install from VSIX myself Oracle extension without finding the problem, but I could not install the oracle plugin correctly from the extension and the following error was reported
![1](https://github.com/user-attachments/assets/a995b037-c653-441c-87af-21a93bd6d3a4)

**Reply by msujew** [reply]
Can you share the backend log? It usually shows a lot more information about the error than the frontend log.
